server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -11,8 +12,16 @@ import (
 	"go.temporal.io/sdk/workflow"
 )
 
-type revisionProcessor interface {
-	Process(ctx workflow.Context, requestedDeployment terraform.DeploymentInfo, latestDeployment *deployment.Info) (*deployment.Info, error)
+type queue interface {
+	Push(terraform.DeploymentInfo)
```

Comment:
nit: I don't think we use the `Push()` method in the queue worker

Reply:
Will remove